src/main/java/de/srendi/advancedperipherals/common/configuration/APConfig.java

```diff
@@ -1,45 +1,53 @@
 package de.srendi.advancedperipherals.common.configuration;
 
-import com.electronwill.nightconfig.core.file.CommentedFileConfig;
 import net.neoforged.fml.ModContainer;
 import net.neoforged.fml.ModLoadingContext;
-import net.neoforged.fml.config.ConfigFileTypeHandler;
-import net.neoforged.fml.config.ModConfig;
+import net.neoforged.fml.config.IConfigSpec;
 import net.neoforged.fml.loading.FMLPaths;
 
-import java.nio.file.Path;
-import java.util.function.Function;
+import java.util.HashMap;
+import java.util.Map;
 
-public class APConfig extends ModConfig {
+public class APConfig {
 
-    public static final ConfigFileHandler CONFIG_FILE_HANDLER = new ConfigFileHandler();
+    // public static final ConfigFileHandler CONFIG_FILE_HANDLER = new ConfigFileHandler();
```

Comment:
Feel free to remove that old stuff, I don't think we need that anymore

I hope the stuff using `APConfigHelper.getAPConfigFilePath` works to change the directory. But we will see when this stuff is buildable